client/src/App.tsx

```diff
@@ -36,12 +36,16 @@ function App(): JSX.Element {
                   <Route exact path="/dashboard">
                     <Dashboard />
                   </Route>
+<<<<<<< HEAD
+                  {/* <Route path="*">
+=======
                   <Route exact path="/notifications">
                     <AllNotifications />
                   </Route>
                   <Route path="*">
+>>>>>>> 40f8899b37730d677da022dde3a6a4526463740f
                     <Redirect to="/login" />
-                  </Route>
+                  </Route> */}
```

Comment:
Let's fix this please, these are merge conflicts and they were not changed properly and would crash our code.

Reply:
Oh, I've fixed it. Thanks